stories/topics/proc-azure-provisioning-aap.adoc

```diff
@@ -32,13 +32,13 @@ The _Administrator Password_ must contain at least 8 characters, and must includ
   * *Managed Resource Group*: A resource group for the managed application infrastructure.
 +
 Keep this resource group isolated from other resource groups, including the _Resource Group_ where you deployed the managed application.
-. Store the information that you entered in the form in a secure place. You must provide the _Administrator password_ to access {ControllerName} and private {HubName}.
-* See the _Change the automation controller Administrator Password_ section of the link:{LinkControllerAdminGuide} guide for help with this.
+. Store the information that you entered in the form in a secure place. You must provide the _Administrator password_ to access your automation platform.
```

Comment:
Use an attribute for Automation platform? And should it be your {PlatformNameShort} instances?